### PR TITLE
Added initial work of statsd exporter plugin.

### PR DESCRIPTION
**Pull request by jeid64**
This adds the start of the statsd exporter plugin to our image. Needs future work to rename all graphite references to statsd.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -162,6 +163,7 @@ RUN /usr/local/bin/install-plugins.sh       \
 # add mesos plugin
 ADD https://infinity-artifacts.s3.amazonaws.com/mesos-jenkins/mesos.hpi-${MESOS_PLUG_HASH} "${JENKINS_STAGING}/plugins/mesos.hpi"
 ADD https://infinity-artifacts.s3.amazonaws.com/prometheus-jenkins/prometheus.hpi-${PROMETHEUS_PLUG_HASH} "${JENKINS_STAGING}/plugins/prometheus.hpi"
+ADD https://infinity-artifacts.s3.amazonaws.com/statsd-jenkins/metrics-graphite.hpi-${STATSD_PLUG_HASH} "${JENKINS_STAGING}/plugins/metrics-graphite.hpi"
```
Comment (benclarkwood) [inline Dockerfile]: I assume there is a Jenkins job for dis now?

Comment (benclarkwood) [reply Dockerfile]: Also, where is the source for this plugin?

File: Dockerfile [hunk 2]
```diff
@@ -16,6 +16,7 @@ ARG BLUEOCEAN_VERSION=1.5.0
 ARG JENKINS_STAGING=/usr/share/jenkins/ref/
 ARG MESOS_PLUG_HASH=5216d7ecef0bc8923ff510aec6659e2c7e7611cb
 ARG PROMETHEUS_PLUG_HASH=a347bf2c63efe59134c15b8ef83a4a1f627e3b5d
+ARG STATSD_PLUG_HASH=statsd
```
Comment (colin-msphere) [inline Dockerfile]: This may not pick up future changes if that hash/tag is update on s3, but cached in docker. Do we want to keep this or use a sha1?

**Comment by jeid64** [comment]
Changed to using https://github.com/mesosphere/jenkins-statsd-plugin
and now uses the git ref.

**Comment by jeid64** [comment]
@mesosphere-ci: re-test this please

**Pull request merged**